dist/src/main/java/io/camunda/application/StandaloneTasklist.java

```diff
@@ -46,6 +48,7 @@
           pattern = "io\\.camunda\\.tasklist\\.archiver\\..*")
     },
     nameGenerator = FullyQualifiedAnnotationBeanNameGenerator.class)
+@Import(WebappsModuleConfiguration.class)
```

Comment:
I suggest to migrate the `StandaloneApplication` to use the `MainSupport` to construct an application as done in `StandaloneOperate`. And the `WebappsModuleConfiguration` should be integrated as an addition `source`.

Reply:
I was planning to make `StandaloneTasklist` use the `MainSupport` in a next PR, but I will do it here if it makes it cleaner
👍  for adding `WebappsModuleConfiguration` in  `StandaloneOperate` and `StandaloneCamunda`